javalee/src/main/java/javalee/com/services/ReturnMaxMinValues.java

```diff
@@ -0,0 +1,35 @@
+package javalee.com.services;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import javalee.com.bd_connection.DbConnection;
+
+public class ReturnMaxMinValues {
+    private Float valorMaximo;
+    private Float valorMinimo;
+
+    public ReturnMaxMinValues(String metrica){
+        DbConnection db = new DbConnection();
+        ResultSet resultMax = db.executeWithReturn("SELECT maximo_risco, minimo_risco FROM metrica WHERE nome = '" + metrica +"'");
+    
+        try {
+            if (resultMax.next()){
+                this.valorMaximo = resultMax.getFloat("maximo_risco");
```

Comment:
Recomendo usar BigDecimal
` this.valorMaximo = BigDecimal.valueOf(resultMax.getDouble("maximo_risco"));
 this.valorMinimo = BigDecimal.valueOf(resultMax.getDouble("minimo_risco"));`
 
 Lembrando que deve ser mudado o tipo do atributo e nos imports deve ser adicionado o 
`import java.math.BigDecimal;`

Reply:
Lindos